cmd/token/list.go

```diff
@@ -87,3 +72,37 @@ func tokenListCmd() *cobra.Command {
 
 	return cmd
 }
+
+func printTokens(writer io.Writer, tokens []token.Token, listTokenRole string) {
+	// Create a metav1.Table object to hold the data
+	table := &metav1.Table{
+		ColumnDefinitions: []metav1.TableColumnDefinition{
+			{Name: "ID", Type: "string", Description: "Token ID"},
+			{Name: "Role", Type: "string", Description: "Token Role"},
+			{Name: "Expires at", Type: "string", Description: "Expiration Time"},
+		},
+	}
+
+	// Populate the rows
+	for _, t := range tokens {
+		if listTokenRole == "" || listTokenRole == t.Role {
+			table.Rows = append(table.Rows, metav1.TableRow{
+				Cells: []interface{}{t.ID, t.Role, t.Expiry},
+			})
+		}
+	}
+
+	// Use a TabWriter for output
+	tabWriter := tabwriter.NewWriter(writer, 0, 0, 2, ' ', 0)
```

Comment:
That looks a bit cryptic :see_no_evil: Wish the upstream API would be a bit more readable.